src/Sylius/Bundle/ChannelBundle/Doctrine/ORM/ChannelRepository.php

```diff
@@ -54,4 +54,9 @@ public function findAllWithBasicData(): iterable
             ->getResult(AbstractQuery::HYDRATE_ARRAY)
         ;
     }
+
+    public function count(array $criteria): int
+    {
+        return parent::count($criteria);
+    }
```

Comment:
I don't know how to do it better. At first sight this is redundant but please see https://github.com/Sylius/Sylius/actions/runs/5046574519/jobs/9052282321

Reply:
or remove typehint from interface

Reply:
👍🏼 for comment, 👎🏼 for removing a type-hint

Reply:
I'm a bit concerned about a possible [interface conflict](https://github.com/doctrine/orm/blob/2.15.x/lib/Doctrine/ORM/EntityRepository.php#L251) in the future due to this.
Doesn't really seem like a pressing matter though

Reply:
I've changed this into `countAll`.